scripts/swap-beacon/src/index.ts

```diff
@@ -246,12 +249,25 @@ export class SwapBeacon {
     );
```

Comment:
could you add a log to state that `convertedOpportunity` with a specific ID was successfully submitted?